kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -19,7 +19,6 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/ui', 'uti
             'error',
             'terminated',
             'does_not_exist',
-            'ee2_error',
```

Comment:
The representation of ee2 errors has changed in backend messages, so all this `ee2_error` error status stuff can be deleted. To do: ensure that the FE can deal with the new way of representing EE2 errors!